pandas/core/indexes/base.py

```diff
@@ -4371,8 +4371,8 @@ def reindex(
 
         See Also
         --------
-        Series.reindex
-        DataFrame.reindex
+        Series.reindex : Return only specified index labels of Series.
```

Comment:
I think these descriptions should be be same except mentioning which object they apply to.

You can copy the first sentence of their docstring:

```
Conform Series/DataFame to new index with optional filling logic.
```

Reply:
I originally took the text from another See Also section. This sounds like a good change, and I've made the update. Thank you for the feedback!